charts/openebs/openebs-target-pod-failure/experiment.yaml

```diff
@@ -11,31 +11,56 @@ metadata:
   version: 0.1.6
 spec:
   definition:
+    scope: Cluster
     permissions:
       - apiGroups:
           - ""
           - "extensions"
```

Comment:
needed?

Reply:
Yes, to get the resource version of the deployment

Reply:
openebs deployments are still using extensions/v1beta1